assets/src/dashboard/app/views/shared/savedTemplateGridView.js

```diff
@@ -0,0 +1,217 @@
+  const containerRef = useRef();
+  const gridRef = useRef();
+  const itemRefs = useRef({});
+  const [activeGridItemId, setActiveGridItemId] = useState(initialFocusId);
+
+  // eslint-disable-next-line no-unused-vars
+  const bottomTargetAction = useCallback(
+    (template) => {
+      return async () => {
+        await trackEvent(
+          'use_saved_template',
+          'dashboard',
+          template.title,
+          template.id
+        );
+        actions.createStoryFromTemplate(template);
+      };
+    },
+    [actions]
+  );
+
+  useGridViewKeys({
+    containerRef,
+    gridRef,
+    itemRefs,
+    isRTL,
+    currentItemId: activeGridItemId,
+    items: templates,
+  });
+
+  // when keyboard focus changes through FocusableGridItem immediately focus the edit preview layer on top of preview
+  useEffect(() => {
+    if (activeGridItemId) {
+      itemRefs.current?.[activeGridItemId]?.children[2].focus();
+    }
+  }, [activeGridItemId]);
+
+  useFocusOut(containerRef, () => setActiveGridItemId(null), []);
+
+  return (
+    <div ref={containerRef}>
+      <StoryGrid
+        pageSize={pageSize}
+        ref={gridRef}
+        role="list"
+        ariaLabel={__('Viewing saved templates', 'web-stories')}
+      >
+        {templates.map((template) => {
+          const isActive = activeGridItemId === template.id;
+          const tabIndex = isActive ? 0 : -1;
+
+          return (
+            <CardGridItem
+              key={template.id}
+              data-testid={`story-grid-item-${template.id}`}
+              role="listitem"
+              ref={(el) => {
+                itemRefs.current[template.id] = el;
+              }}
+            >
+              <FocusableGridItem
+                onFocus={() => {
+                  setActiveGridItemId(template.id);
+                }}
+                isSelected={isActive}
+                tabIndex={tabIndex}
+                title={sprintf(
+                  /* translators: %s: saved template title.*/
+                  __('Press Enter to explore details about %s', 'web-stories'),
+                  template.title
+                )}
+              />
+              <CardPreviewContainer
+                ariaLabel={sprintf(
+                  /* translators: %s: saved template title. */
+                  __('preview of %s', 'web-stories'),
+                  template.title
+                )}
+                tabIndex={tabIndex}
+                pageSize={pageSize}
+                story={template}
+                centerAction={{
+                  targetAction: (e) => actions.previewTemplate(e, template),
+                  label: centerActionLabelByStatus[template.status],
+                }}
+                bottomAction={{
+                  targetAction: false, // bottomTargetAction(template), this doesn't work yet and will throw an error in the editor, commenting out to prevent headaches while testing UI but leaving code in so that we know where things attach
```

Comment:
Since this propType requires a function or nothing we should probably provide an empty function or `undefined`

Reply:
Since this isn't functional yet rather than showing a disabled action I opted to not pass in bottomAction for this grid and update the card item preview to check if bottomAction exists, when it doesn't we'll not render it. I think this keeps the temporary-in-progress state of this page more apparent.